templates/web/diff.ftl

```diff
@@ -108,27 +108,30 @@
   </div>
 </#if>
 
+<#include "/static-assets/app/pages/legacy.html">
 <script>
-  $(document).ready(function(){
-    CStudioAuthoring.Service.getCurrentVersion(CStudioAuthoringContext.site, "${dir}", {
-      success: function(version) {
-        $('#current-version').append(version)
-      }
-    });
+  document.addEventListener("CrafterCMS.CodebaseBridgeReady", () => {
+    jQuery(function() {
```

Comment:
Probably not necessary to add the jquery ready function

Reply:
Yeah it wasn't necessary, though it seems like it was delaying the script and causing the store to be ready, so I removed the jquery ready and now I subscribed to the store for it to be ready when calling the request (for it to have the needed tokens, etc)